Sources/Async/BackendService+Async.swift

```diff
@@ -16,7 +16,7 @@ extension BackendServiceProtocol {
     /// - Parameters:
     ///   - request: The Request to be executed.
     /// - Returns: The decodable type specified in the `Request`.
-    public func execute<T, U>(request: Request<T, U>) async throws -> T {
+    public func executeAsync<T, U>(request: Request<T, U>) async throws -> T {
```

Comment:
Should we rename `executeWithResult` similarly (even though it doesn't have a similar naming issue currently) ?

Reply:
Ugh, nevermind. Gonna delete this PR because I'm a goober who tried to implement the `async execute` wrong.

There's no namespace problem, it was an error I should have encountered earlier with an improperly defined `Request`. 

It's all working now.